src/view/directives/position.js

```diff
@@ -71,8 +72,12 @@ class Position extends Directive(POSITION_NAME, {root: `^?${GRID_NAME}`}) {
 		const intersection = max(intersections, i => i.area);
 		const rect = intersection.b;
 
-		source.style.left = tr.left + (rect.left - tr.left) + 'px';
-		source.style.top = tr.top + (rect.top - tr.top) + 'px';
+		const left = tr.left + (rect.left - tr.left);
```

Comment:
Hmm...I tried to open brackets and looks that something is redundant :)

Reply:
fuck yeah)

Reply:
actually this will be not the last version of layout algo)

Reply:
I suppose it should be updated both here and in #482